MonoGame.Framework/Graphics/GraphicsAdapter.cs

```diff
@@ -19,13 +19,25 @@ namespace Microsoft.Xna.Framework.Graphics
 {
     public sealed class GraphicsAdapter : IDisposable
     {
-
+        /// <summary>
+        /// Defines the driver type for graphics adapter.
+        /// </summary>
         public enum DriverType
         {
+            /// <summary>
+            /// Hardware device been used for rendering. Maximum speed and performance.
+            /// </summary>
             Hardware,
+            /// <summary>
+            /// Emulates the hardware device on CPU. Slowly, only for testing.
+            /// </summary>
             Reference,
+            /// <summary>
+            /// Useful when <see cref="DriverType.Hardware"/> acceleration does not work.
+            /// </summary>

```

Comment:
We should note that `Reference` and `FastSoftware` only work on DirectX platforms at the moment.  Not sure if we're better off listing specific platforms it works on like "Windows desktop and Windows Store".

Don't think those work on Windows Phone... someone needs to check.